### DISCUSSION TITLE
"Soft Wrap" set as default?

### DISCUSSION THREAD

**Discussion by Daibhre** [atom/atom#23284] [General]
I'm a user (not a coder) and I really like using Atom as an HTML editor.
HOWEVER, every time I open a document, I have to click the "Toggle Soft Wrap" in the menu to see all the code.
Can that be set to "ON" as a default?
If not, I'll make a keyboard shortcut - but I'd rather be lazy and just have it always set to ON.

**Comment by CoderKeung** [comment]
https://flight-manual.atom.io/getting-started/sections/atom-basics/#soft-wrap

This might help you.

**Comment by Daibhre** [comment]
Many thanks!

- David M.
www.dmccord.com / www.newventuregames.com
https://www.facebook.com/NewVentureGames
